vendor/engines/secure_rooms/spec/services/secure_rooms/access_handlers/occupancy_handler_spec.rb

```diff
@@ -30,15 +30,33 @@
     end
   end
 
+  shared_examples_for "a problem order" do
+    it "sets up a problem order" do
+      expect(occupancy.order_detail).to be_complete
+      expect(occupancy.order_detail).to be_problem
+    end
+  end
+
   describe "#process" do
     context "current_occupant?" do
+      let(:order) do
```

Comment:
Stylistically, I prefer `let`s to use `{ }`, even if it results in a slightly longer line. But that's also because I'm not a stickler for line length (well, until it gets 100+).